src/Evolution/Executables/GrMhd/ValenciaDivClean/EvolveValenciaDivClean.hpp

```diff
@@ -131,30 +132,7 @@ class CProxy_ConstGlobalCache;
 }  // namespace Parallel
 /// \endcond
 
-struct KerrHorizon {
-  using tags_to_observe =
-      tmpl::list<StrahlkorperTags::EuclideanSurfaceIntegralVector<
-          hydro::Tags::MassFlux<DataVector, 3, ::Frame::Inertial>,
-          ::Frame::Inertial>>;
-  using compute_items_on_source = tmpl::list<>;
-  using vars_to_interpolate_to_target = tmpl::list<
-      hydro::Tags::RestMassDensity<DataVector>,
-    hydro::Tags::SpatialVelocity<DataVector, 3, Frame::Inertial>,
-      hydro::Tags::LorentzFactor<DataVector>, gr::Tags::Lapse<DataVector>,
-      gr::Tags::Shift<3, Frame::Inertial, DataVector>,
-      gr::Tags::SqrtDetSpatialMetric<DataVector>>;
-  using compute_items_on_target = tmpl::push_front<
-      tags_to_observe,
-      StrahlkorperTags::EuclideanAreaElement<::Frame::Inertial>,
-      hydro::Tags::MassFluxCompute<DataVector, 3, ::Frame::Inertial>>;
-  using compute_target_points =
-      intrp::TargetPoints::KerrHorizon<KerrHorizon, ::Frame::Inertial>;
-  using post_interpolation_callback =
-      intrp::callbacks::ObserveTimeSeriesOnSurface<tags_to_observe, KerrHorizon,
-                                                   KerrHorizon>;
-};
-
-template <typename InitialData, typename...InterpolationTargetTags>
+template <typename InitialData, typename... InterpolationTargetTags>
```

Comment:
Why did you move this struct? Just wondering

Reply:
The original version of getting the interpolation points to the elements required passing around the pdal of the dg element array, which I thought was inelegent, so I've added the type alias `interpolating_dg_component` to the interpolation tags that would need access to that type. However, that new requires the dg element array component type to exist before I define the interpolation tag struct, so it forced the struct definition to follow the metavariables definition.